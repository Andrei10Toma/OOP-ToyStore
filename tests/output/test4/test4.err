exceptions.CommandNotFound: Input command listmannufacturers not found.
	at main.Main.interpretCommand(Main.java:69)
	at main.Main.main(Main.java:35)
exceptions.ProductNotFound: Product with id aaaaaa not found.
	at command.ShowProduct.execute(ShowProduct.java:28)
	at command.CommandQueue.executeCommand(CommandQueue.java:14)
	at main.Main.interpretCommand(Main.java:47)
	at main.Main.main(Main.java:35)
exceptions.DiscountNotFoundException: Discount PERCENTAGE_DISCOUNT 45.0 not found.
	at command.ApplyDiscount.execute(ApplyDiscount.java:35)
	at command.CommandQueue.executeCommand(CommandQueue.java:14)
	at main.Main.interpretCommand(Main.java:47)
	at main.Main.main(Main.java:35)
exceptions.NegativePriceException: The product with price 7.5075 and id 95ddb46815960a546e22a653fbb300e8 cannot have a fixed discount of 8.0 applied.
	at store.Store.applyDiscountProducts(Store.java:145)
	at command.ApplyDiscount.execute(ApplyDiscount.java:37)
	at command.CommandQueue.executeCommand(CommandQueue.java:14)
	at main.Main.interpretCommand(Main.java:47)
	at main.Main.main(Main.java:35)
exceptions.ManufacturerNotFound: Manufacturer with name aaaaaa bbbbbb not found.
	at command.ListProductsByManufacturer.execute(ListProductsByManufacturer.java:36)
	at command.CommandQueue.executeCommand(CommandQueue.java:14)
	at main.Main.interpretCommand(Main.java:47)
	at main.Main.main(Main.java:35)
