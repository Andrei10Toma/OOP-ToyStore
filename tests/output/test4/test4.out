EUR
EUR 1.00
GBP 1.10
RON 0.20
USD 0.82
PERCENTAGE_DISCOUNT 50.0 "EVERYTHING AT HALF PRICE" 2021-01-18T13:32:06
PERCENTAGE_DISCOUNT 30.0 "WINTER SALE" 2021-01-18T13:32:06
FIXED_DISCOUNT 8.0 "-8 AT EVERYTHING" 2021-01-18T13:32:06
EUR
RUB
4eb5ea748ea030b14c3d1789f9578133,Plarail - S-29 Steam Locomotive Type C61-20 w/Head Light (Model Train),Takara Tomy,₽900.900,37
6c4062e5b0da136365915b470ec9d4a4,Plarail - AS-07 Shinkansen Series 700 (Model Train),Takara Tomy,₽1,010.800,35
660b30ec7520119f22681588b9b1e25e,S-60 EF66 Electric Locomotive (Tomica PlaRail Model Train) (japan import),Takara Tomy,₽810.600,31
1150534cee9ef4fa07f22dd1c709866d,Plarail - S-24 Series 485 Limited Express Train (Model Train),Takara Tomy,₽775.600,38
d0751cce2a2d8e32582c850d266cf93d,Takara Tomy Tomica #069 Nissan Diesel Quon Aquarium Truck,Takara Tomy,₽224.000,35
e781c1d9e9534c1c97733e34f432f969,S-59 Keihan 8000 Express (Tomica PlaRail Model Train) (japan import),Takara Tomy,₽1,313.200,36
1d865c2b765ea2b40ad2abd46889aefc,Takaratomy 819738 Official Pokemon X and Y MC-056 2" Treecko Action Figure,Takara Tomy,₽489.300,20
4300c37e68cf3aaa9b0f0d25c10751ce,Lucario Pokemon Pocket Monster Collection Takara Tomy Action Figure,Takara Tomy,₽296.100,37
9fa0d2994fbeede49433140da0fd9e6e,Transformers Binaltech Bt-21 Hardtop Featuring Arcee,Takara Tomy,₽3,204.600,8
5e35b3a765fb26a01e702573ecf680d9,Optimus Prime DMK 01 Transformers Movie Dual Model Kit,Takara Tomy,₽6,227.200,18
8264b8217b28f09e6dc85731daa6969c,Takara Tomy Beyblades #Bb83 Japanese Metal Fusion Df145Bs Premium Returns Booster Pisces Battle Top,Takara Tomy,₽447.300,30
dc882b5d3fc71fb4d970733742be4f93,Tomica No.50 - LOTUS EXIGE S (Box),Takara Tomy,₽328.300,30
3cf940fceb0234e007da0707d4bb7c9a,Tomica Town Gasoline Station (ENEOS) from Japan,Takara Tomy,₽1,050.000,29
31b78ffc4162110569c34fdd331e2703,Transformers QT22 Drift (Mazda Rx-7 FD3S),Takara Tomy,₽497.700,17
0034d77b68130bba71aca6e733c7e1ac,Pokemon Black and White Voice Activated Talking Plush Figure - 10" Mijumaru / Oshawott,Takara Tomy,₽2,116.100,3
a6c94525c824d2519ee8bc9de2f1b615,Pokemon: Best Wishes! (Black + White) Soft Toy Plush Figure: Zorua (laughing) 20 cm (T.T.),Takara Tomy,₽1,749.300,17
ea46926383b95214e6dd9652f70e8322,S-01 Bullet Train Series 700 with Headlight,Takara Tomy,₽1,215.200,26
7076dded059ea01b8605f532c704a539,AS-04 Series E231-500 Yamanote Line,Takara Tomy,₽942.900,24
bd62dc6dd6bb5af7835be4f6b7321bf1,Plarail - S-47 Sound Enoden Type1000 (Update Color) (Model Train),Takara Tomy,₽829.500,31
443bdf2c385b4480d75c746a0f3c6e32,Takara Tomy Tomica #015 Hummer H2,Takara Tomy,₽256.200,35
5a48b39cd9e1f797dee642cc95bac2ff,King Kong - Roaring Bull V-Rex,Takara Tomy,₽15,097.600,5
062b36a01be7a910c555a7a5880cab3d,Optimus Prime DMK 01 Transformers Movie Dual Model Kit,Takara Tomy,₽7,602.000,2
2d1d0e764434936c14cdb6d50782a969,Takara Tomy Beyblades Japanese Metal Fusion Starter Set #Bb106 Fang Leone,Takara Tomy,₽2,885.400,16
84d213f126cbc9b329d356dc990fa13a,Takara Tomy Beyblades Japanese Zero G #Bbg01 Samurai Ifraid Starter Set,Takara Tomy,₽590.100,27
b8ae267aaaa750e1d5c49325d280a314,Beyblade A-78 Gaia Dragoon V,Takara Tomy,₽1,312.500,24
923774e4be6a0e62612d7292d7ed115b,Takara Tomy Beyblades Japanese Metal Fusion Battle Top Starter #Bb47 Earth Eagle 145Wd Includes Light Launcher!,Takara Tomy,₽3,186.400,20
87c2e4a5984ea5424220a3ce015f0f2f,Takara Tomy Pokemon Monster Collection Mini Figure - 1.5" Zapdos / Thunder (M-064) (Japanese Import),Takara Tomy,₽2,319.800,7
4ae5dd2fd1441ba1aefcd8b4fd3111ab,POKEMON MONSTER COLLECTION 1.5" FIGURE MC-37 ARTICUNO,Takara Tomy,₽737.800,24
578c56f22fd9408e135fce6d32d7712a,Tomikakazu C-24 flow (standard type) (japan import),Takara Tomy,₽366.800,32
082a03eb68b2083154b2edc2e1196875,The Railway Collection Toyama Light Rail TLR0607 (Purple),Takara Tomy,₽994.700,14
c85169bcc88f9ee8e83e2899afae3ba6,Plarail - Thomas & Friends: Hiro and Aquarium Car Set (Model Train),Takara Tomy,₽1,258.600,23
e0bbdea0003ef69c431effaf5505461d,Plarail - Series E6 [Super Komachi] & Tomica Arch Rail Crossing Set (Model Train),Takara Tomy,₽3,922.800,21
b80cea4a2bfb7273e62eab3157fb721b,S-39 EF510 Red Thunder (Tomica PlaRail Model Train) [Toy],Takara Tomy,₽795.200,38
6ba28b725623960116677993bc1e60ce,Carriages and two Tomica Thomas Henry (japan import),Takara Tomy,₽945.000,18
c96734af2f2f6b55057673d8e0688f64,Takara Tomy Pokemon Monster Collection SP-15 Mega Charizard X,Takara Tomy,₽406.700,30
02db02d58b42826cea6bde42dd057393,Poket Monster Doll Japan National Football Team with Pikachu Flag,Takara Tomy,₽1,355.200,27
8cdaf5c22e1b72afd55c33b46d2145c7,Beyblade Big Bang Pegasus 4D (Cosmic Pegasus) - Season 3 Beyblade Metal Fury 4D -,Takara Tomy,₽1,400.000,2
d3b16807bcb284e42c2a66f97a27a2a0,Official Nintendo Pokemon Diamond & Pearl Plush Hand Puppet - 14" Chimchar (Japanese Import),Takara Tomy,₽2,713.200,5
d92fdf9384e7d5227c86411ad391abf6,Transformers Go! EG10 jet beacon (japan import),Takara Tomy,₽1,346.100,12
3a497a8b7edffa5fa720ea6568e07e06,Transformers Japanese Animated Wingblade Optimus Prime TA-38(Japan import),Takara Tomy,₽4,590.600,22
777cdc8ff911615b926d57ecf1a31a66,Takara Tomy Pokemon Monster Collection Mini Figure - 1.5" Empoleon / Emperte (M-119) (Japanese Import),Takara Tomy,₽987.700,7
EUR
PERCENTAGE_DISCOUNT 50.0 "EVERYTHING AT HALF PRICE" 2021-01-18T13:32:06
PERCENTAGE_DISCOUNT 30.0 "WINTER SALE" 2021-01-18T13:32:06
FIXED_DISCOUNT 8.0 "-8 AT EVERYTHING" 2021-01-18T13:32:06
4eb5ea748ea030b14c3d1789f9578133,Plarail - S-29 Steam Locomotive Type C61-20 w/Head Light (Model Train),Takara Tomy,€14.157,37
6c4062e5b0da136365915b470ec9d4a4,Plarail - AS-07 Shinkansen Series 700 (Model Train),Takara Tomy,€15.884,35
660b30ec7520119f22681588b9b1e25e,S-60 EF66 Electric Locomotive (Tomica PlaRail Model Train) (japan import),Takara Tomy,€12.738,31
1150534cee9ef4fa07f22dd1c709866d,Plarail - S-24 Series 485 Limited Express Train (Model Train),Takara Tomy,€12.188,38
d0751cce2a2d8e32582c850d266cf93d,Takara Tomy Tomica #069 Nissan Diesel Quon Aquarium Truck,Takara Tomy,€3.520,35
e781c1d9e9534c1c97733e34f432f969,S-59 Keihan 8000 Express (Tomica PlaRail Model Train) (japan import),Takara Tomy,€20.636,36
1d865c2b765ea2b40ad2abd46889aefc,Takaratomy 819738 Official Pokemon X and Y MC-056 2" Treecko Action Figure,Takara Tomy,€7.689,20
4300c37e68cf3aaa9b0f0d25c10751ce,Lucario Pokemon Pocket Monster Collection Takara Tomy Action Figure,Takara Tomy,€4.653,37
9fa0d2994fbeede49433140da0fd9e6e,Transformers Binaltech Bt-21 Hardtop Featuring Arcee,Takara Tomy,€50.358,8
5e35b3a765fb26a01e702573ecf680d9,Optimus Prime DMK 01 Transformers Movie Dual Model Kit,Takara Tomy,€97.856,18
8264b8217b28f09e6dc85731daa6969c,Takara Tomy Beyblades #Bb83 Japanese Metal Fusion Df145Bs Premium Returns Booster Pisces Battle Top,Takara Tomy,€7.029,30
dc882b5d3fc71fb4d970733742be4f93,Tomica No.50 - LOTUS EXIGE S (Box),Takara Tomy,€5.159,30
3cf940fceb0234e007da0707d4bb7c9a,Tomica Town Gasoline Station (ENEOS) from Japan,Takara Tomy,€16.500,29
31b78ffc4162110569c34fdd331e2703,Transformers QT22 Drift (Mazda Rx-7 FD3S),Takara Tomy,€7.821,17
0034d77b68130bba71aca6e733c7e1ac,Pokemon Black and White Voice Activated Talking Plush Figure - 10" Mijumaru / Oshawott,Takara Tomy,€33.253,3
a6c94525c824d2519ee8bc9de2f1b615,Pokemon: Best Wishes! (Black + White) Soft Toy Plush Figure: Zorua (laughing) 20 cm (T.T.),Takara Tomy,€27.489,17
ea46926383b95214e6dd9652f70e8322,S-01 Bullet Train Series 700 with Headlight,Takara Tomy,€19.096,26
7076dded059ea01b8605f532c704a539,AS-04 Series E231-500 Yamanote Line,Takara Tomy,€14.817,24
bd62dc6dd6bb5af7835be4f6b7321bf1,Plarail - S-47 Sound Enoden Type1000 (Update Color) (Model Train),Takara Tomy,€13.035,31
443bdf2c385b4480d75c746a0f3c6e32,Takara Tomy Tomica #015 Hummer H2,Takara Tomy,€4.026,35
5a48b39cd9e1f797dee642cc95bac2ff,King Kong - Roaring Bull V-Rex,Takara Tomy,€237.248,5
062b36a01be7a910c555a7a5880cab3d,Optimus Prime DMK 01 Transformers Movie Dual Model Kit,Takara Tomy,€119.460,2
2d1d0e764434936c14cdb6d50782a969,Takara Tomy Beyblades Japanese Metal Fusion Starter Set #Bb106 Fang Leone,Takara Tomy,€45.342,16
84d213f126cbc9b329d356dc990fa13a,Takara Tomy Beyblades Japanese Zero G #Bbg01 Samurai Ifraid Starter Set,Takara Tomy,€9.273,27
b8ae267aaaa750e1d5c49325d280a314,Beyblade A-78 Gaia Dragoon V,Takara Tomy,€20.625,24
923774e4be6a0e62612d7292d7ed115b,Takara Tomy Beyblades Japanese Metal Fusion Battle Top Starter #Bb47 Earth Eagle 145Wd Includes Light Launcher!,Takara Tomy,€50.072,20
87c2e4a5984ea5424220a3ce015f0f2f,Takara Tomy Pokemon Monster Collection Mini Figure - 1.5" Zapdos / Thunder (M-064) (Japanese Import),Takara Tomy,€36.454,7
4ae5dd2fd1441ba1aefcd8b4fd3111ab,POKEMON MONSTER COLLECTION 1.5" FIGURE MC-37 ARTICUNO,Takara Tomy,€11.594,24
578c56f22fd9408e135fce6d32d7712a,Tomikakazu C-24 flow (standard type) (japan import),Takara Tomy,€5.764,32
082a03eb68b2083154b2edc2e1196875,The Railway Collection Toyama Light Rail TLR0607 (Purple),Takara Tomy,€15.631,14
c85169bcc88f9ee8e83e2899afae3ba6,Plarail - Thomas & Friends: Hiro and Aquarium Car Set (Model Train),Takara Tomy,€19.778,23
e0bbdea0003ef69c431effaf5505461d,Plarail - Series E6 [Super Komachi] & Tomica Arch Rail Crossing Set (Model Train),Takara Tomy,€61.644,21
b80cea4a2bfb7273e62eab3157fb721b,S-39 EF510 Red Thunder (Tomica PlaRail Model Train) [Toy],Takara Tomy,€12.496,38
6ba28b725623960116677993bc1e60ce,Carriages and two Tomica Thomas Henry (japan import),Takara Tomy,€14.850,18
c96734af2f2f6b55057673d8e0688f64,Takara Tomy Pokemon Monster Collection SP-15 Mega Charizard X,Takara Tomy,€6.391,30
02db02d58b42826cea6bde42dd057393,Poket Monster Doll Japan National Football Team with Pikachu Flag,Takara Tomy,€21.296,27
8cdaf5c22e1b72afd55c33b46d2145c7,Beyblade Big Bang Pegasus 4D (Cosmic Pegasus) - Season 3 Beyblade Metal Fury 4D -,Takara Tomy,€22.000,2
d3b16807bcb284e42c2a66f97a27a2a0,Official Nintendo Pokemon Diamond & Pearl Plush Hand Puppet - 14" Chimchar (Japanese Import),Takara Tomy,€42.636,5
d92fdf9384e7d5227c86411ad391abf6,Transformers Go! EG10 jet beacon (japan import),Takara Tomy,€21.153,12
3a497a8b7edffa5fa720ea6568e07e06,Transformers Japanese Animated Wingblade Optimus Prime TA-38(Japan import),Takara Tomy,€72.138,22
777cdc8ff911615b926d57ecf1a31a66,Takara Tomy Pokemon Monster Collection Mini Figure - 1.5" Empoleon / Emperte (M-119) (Japanese Import),Takara Tomy,€15.521,7
